RingTask/Classes/Flow/RedditFlow/RedditFlowCoordinator.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RedditFlowCoordinator.swift
+//  RingTask
+//
+//  Created by Hryhorii Maksiuk on 8/7/17.
+//  Copyright © 2017 Gregory M. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+
+class RedditFlowCoordinator: NSObject {
```

Comment:
WTF at first look. Then it become clear it is required by SafariVC. Can it be conformed to `NSObject` in appropriate extension?